### ISSUE TITLE
Issue with ModManager reading mod.ini in the mods folder 

### ISSUE THREAD

**Issue by thesupersonic16** [sonicretro/mania-mod-loader#10]
I haven't written an issue in a very long time.

So it looks like ManiaModManager is trying to load mod.ini from `Sonic Mania/mods/mod.ini` and is failing to substring the path.

The issue is that the ModManager is failing to substring the path for the key because its trying to get the directory name by removing the mods directory path plus a slash(I guess), which will not work if the mod.ini is in the mods directory, The exception is thrown at [line 217 in MainForm.cs](https://github.com/sonicretro/mania-mod-loader/blob/master/ManiaModManager/MainForm.cs#L217) when mod.ini exists in the mods directory.

I have no idea if you wanted the mods directory to be a mod.

**Comment by MainMemory** [comment]
No, there shouldn't be a mod.ini file in the mods folder. I don't know where you're getting one from, but it's not included with the mod loader.

**Comment by thesupersonic16** [comment]
Its not included, but some people are extracting mod.ini in their mods folder which causes the ModManager to throw an exception.

**Comment by MainMemory** [comment]
Well then people should include the folder in their mod packages, like I do.

I guess I could try to add some kind of warning about having mod.ini in the mods folder.

**Comment by thesupersonic16** [comment]
That should help.
